guidelines/supportability.md

```diff
@@ -25,20 +25,26 @@ There are significant risks associated with exposing customers to Alpha-level up
 * The api/feature may change in a way that we cannot migrate customer workloads/configuration/resources
 * The api/feature may be removed entirely in a future k8s version leaving us having to carry patches or manually help customers to move forward which may even require an entirely new cluster installation
 
-Given these risks, alpha features **MUST NOT** be turned on downstream unless behind a feature gate that can only be enabled via the TechPreviewNoUpgrade 
-or CustomNoUpgrade featuresets.  If you/your team intend to enable (without a TechPreview gate) an upstream alpha feature, downstream, you must bring the request to 
-the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls (this may not get you approval 
-but it is a good place to start the conversation about what you want to do and why), or tagging `@aos-staff-engineers` on `#forum-arch`.  Approval won't be granted 
+Given these risks, alpha features **MUST NOT** be turned on downstream unless behind a feature gate that can only be enabled via the TechPreviewNoUpgrade
+or CustomNoUpgrade featuresets.  If you/your team intend to enable (without a TechPreview gate) an upstream alpha feature, downstream, you must bring the request to
+the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls (this may not get you approval
+but it is a good place to start the conversation about what you want to do and why), or tagging `@aos-staff-engineers` on `#forum-arch`.  Approval won't be granted
 without a set of people who clearly own working on the feature upstream and ensuring it will be driven to beta/GA state.
 
 Enabling an upstream alpha feature by putting it behind a TechPreviewNoUpgrade feature gate is acceptable as long as the team that is enabling it understands
 they are going to be responsible for helping debug issues associated with it.
 
 ## Enabling Upstream Beta features
 
-TBD - we'd like to disable these by default(and then have a relatively lower bar for enabling them), but right no there's no good way to disable the associated 
+TBD - we'd like to disable these by default (and then have a relatively lower bar for enabling them), but right no there's no good way to disable the associated
```

Comment:
you can take out the TBD, the exercise you're doing now is the "determination" that was previously "to be determined".

but keeping the context about how we are constrained in terms of disabling upstream beta features is still good context (and making it clear the constraint might go away in the future, at which point we'd have more options in terms of choosing to disable upstream beta features)

And again i think everywhere in this section we need to be very clear about whether we're talking about a beta feature that's tied to a beta api, or one that's not tied to a beta api, since that has significant implications on whether it can be enabled by default upstream or not, as well as whether/how our downstream users can be expected to understand whether it's a beta(subject to changing/going away) or not.